docs/administration/garbage-collection/_index.md

```diff
@@ -37,7 +38,8 @@ You are able to schedule garbage colleciton on your Harbor interface from the **
     * **Weekly**: Run garbage collection at midnight every Saturday.
     * **Custom**: Run garbage collection according to a `cron` job.
 
-1. To **Allow garbage collection on untagged artifacts**, select the option. If you have this option turned on, the next time Garbage collect runs on your Harbor instance, Harbor will delete untagged artifacts and then perform garbage collection on them.     
+1. To **Workers**, select the option. Set the number of workers that can execute GC tasks in parallel, the default value is 1.
```

Comment:
What about `For the Workers item, you are able to select the number of workers that can execute GC tasks in parallel`?